src/main/java/sirius/biz/storage/layer3/VirtualFileSystemController.java

```diff
@@ -222,6 +223,36 @@ public void delete(WebContext webContext) {
         }
     }
 
+    /**
+     * Deletes the given list of files or directories.
+     *
+     * @param webContext the request to handle
+     */
+    @LoginRequired
+    @Routed("/fs/delete/multiple")
+    @Permission(PERMISSION_VIEW_FILES)
+    public void deleteMultiple(WebContext webContext) {
```

Comment:
Does this respect whether or not the selected paths are deletable?
In the template the delete action for a single file row is restricted via `child.canDelete()`.
How does it behave when a not deletable file or sub-folder is included in the selection?
Same question for the move operation